boards/firefly/include/board.h

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (C) 2017 DAI Labor TU Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @defgroup    boards_firefly Firefly Board
+ * @ingroup     boards
+ * @brief       Support for the Firefly board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Firefly board
+ *
+ * @author      Anon Mall <anon.mall@gt-arc.com>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "board_common.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+ extern "C" {
+#endif
+
+/**
+ * @name LED pin definitions
+ * @{
+ */
+#define LED0_PIN        GPIO_PIN(3, 5)
```

Comment:
Is there an enum for the port ? something like PORT_A/B/C/D ?
It would be great to add a define for the port of each LED/BTN
```c
#define LED0_PORT PORT_D
```
for example

Reply:
the cc2538 gpio code is currently in change, the port enums are flagged as deprecated. See #7517

Reply:
Ah I actually just found the enums in the CC2538. I missed that. I will change it no problem